TwoSum/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TwoSum
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Test Case 1
+            int[] nums1 = { 2, 7, 11, 15 };
+            int targets1 = 9;
+            int[] result1 = TwoSumEqualTarget(nums1, targets1);

```

Comment:
General comment for the whole file, we can declare variables implicitly in C#, which is `var` instead of writing the type of the variables,
Read this: https://www.dotnettricks.com/learn/csharp/c-sharp-var-data-type-and-anonymous-type